posts/a-devs-intro-to-devops.md

```diff
@@ -0,0 +1,46 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- DevOps
+- AWS GovCloud
+- OpenShift
+- Infrastructure as Code
+- IaC
+date: 2015-04-23T12:21:50.000Z
+title: "A Dev's Intro to DevOps"
+image: 
+---
+I have recently been sucked into the field known as DevOps and I thought it would be fun to share my story. I have spent most of my career in the application development space. I am curious if others have had similar experiences.
```

Comment:
I have recently been sucked into the field known as "DevOps" and I thought it would be fun to share my story. Having spent most of my career in application development, I am curious to see if others have had a similar experience to mine.